mlflow/server/js/src/components/MetricsPlotPanel.js

```diff
@@ -260,6 +260,7 @@ export class MetricsPlotPanel extends React.Component {
    * and schema.
    */
   handleLayoutChange = (newLayout) => {
+    this.isClicked = false;
```

Comment:
I haven't yet figured out what causes this bug, but setting `isClicked` to `false` in `handleLayoutChange` fixes the bug.